src/main/kotlin/xyz/atrius/waystones/commands/WarpstoneCommand.kt

```diff
@@ -0,0 +1,71 @@
+package xyz.atrius.waystones.commands
+
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import org.bukkit.Bukkit
+import xyz.atrius.waystones.utility.defaultWarpKey
+import xyz.atrius.waystones.utility.translateColors
+import xyz.atrius.waystones.utility.pluralS
+
+object WarpstoneCommand : CommandExecutor {
+    // CommandExecutor / Subcommand Manager
+    override fun onCommand(
+        sender: CommandSender,
+        cmd: Command,
+        lbl: String,
+        args: Array<String>
+    ): Boolean {
+        // Plugin Info
+        if (args.isEmpty())
+            return infoCmd(sender)
+
+        // Register Give WarpKey
+        val getkeyAliases = arrayOf("getkey", "gk", "warpkey", "wk", "key", "k")
+        if (args[0].toLowerCase() in getkeyAliases)
+            return getkeyCmd(sender, args.drop(1))
+
+        // Command Not Found
+        else sender.sendMessage("&d[Waystones]&r Unknown command, check &a/waystones&r".translateColors('&'))
+        return true
+    }
+
+    // Command - Plugin Info
+    private fun infoCmd (sender: CommandSender): Boolean {
+        val plugin = Bukkit.getPluginManager().getPlugin("Waystones")
```

Comment:
what is this being used for?

Reply:
The version reference a few lines below, although I imagine there to be some null issues regarding that, since `.getPlugin()` could return null.

But it hasn't failed thus far and if it does (I used the wrong plugin name initially, lol) the version number will just be blank, if I remember correctly.

Reply:
Ah I see now, there's already an instance of the plugin in `Waystones.kt`. Just import `plugin` and you should be able to reference it that way.

Reply:
Fair point, I didn't realize I could access that so easily. Thanks for the heads up, I've reworked it.